charts/vela-core/templates/defwithtemplate/apply-component.yaml

```diff
@@ -0,0 +1,21 @@
+# Code generated by KubeVela templates. DO NOT EDIT. Please edit the original cue file.
+# Definition source cue file: vela-templates/definitions/internal/apply-component.cue
+apiVersion: core.oam.dev/v1beta1
+kind: WorkflowStepDefinition
+metadata:
+  annotations:
+    definition.oam.dev/description: Apply a component and its corresbonding traits in application
+  labels:
+    custom.definition.oam.dev/scope: Application
+    custom.definition.oam.dev/ui-hidden: "true"
+  name: apply-component
+  namespace: {{ include "systemDefinitionNamespace" . }}
+spec:
+  schematic:
+    cue:
+      template: |
+        parameter: {
+        	// +usage=Specify the component name to apply
+        	component: string
```

Comment:
apply component also support specify cluster?

Reply:
No, `cluster` is valid for `op.#ApplyComponent` but not `apply-component` def.